src/components/ThemesSection.tsx

```diff
@@ -79,9 +79,9 @@ const ThemesSection = ({ maxThemes }: { maxThemes?: number }) => {
           <a
             key={theme.name}
             className="bg-white py-6 flex items-center justify-center px-2 rounded-lg h-[166px] shadow-lg border-b-4 border-b-[#B5BFC4] hover:border-b-secondary transition hover:bg-gray-50"
-            href={`/datasets?group=${theme.name
+            href={`/datasets?page=1&ckan-theme=${(theme.desc || theme.name)
```

Comment:
important: Can you please reuse a function to support decoding text to URL? In JavaScript there a few ways to do that.

Reply:
I will do them in the next pr when I add the URI's